src/utils/update/beneficial_owners_managing_officers_data_fetch.ts

```diff
@@ -34,21 +35,27 @@ const initialiseBoAndMoUpdateAppData = (appData: ApplicationData) => {
 };
 
 export const retrieveBeneficialOwners = async (req: Request, appData: ApplicationData) => {
+  const transactionId = appData.transaction_id;
+  const overseasEntityId = appData.overseas_entity_id;
   const pscs: CompanyPersonsWithSignificantControl = await getCompanyPsc(req, appData[EntityNumberKey] as string);
+  const boPrivateData: BeneficialOwnersPrivateDataResource = await getBeneficialOwnerPrivateData(req, transactionId, overseasEntityId) as BeneficialOwnersPrivateDataResource;
   if (pscs) {
     for (const psc of (pscs.items || [])) {
       logger.info("Loaded psc " + psc.kind);
       if (psc.ceasedOn === undefined) {
         if (psc.kind === "individual-beneficial-owner") {
           const individualBeneficialOwner = mapPscToBeneficialOwnerTypeIndividual(psc);
+          individualBeneficialOwner.usual_residential_address = mapPrivateAddress(boPrivateData, individualBeneficialOwner.ch_reference as string);
           logger.info("Loaded individual Beneficial Owner " + individualBeneficialOwner.id + " is " + individualBeneficialOwner.first_name + ", " + individualBeneficialOwner.last_name);
           appData.update?.review_beneficial_owners_individual?.push(individualBeneficialOwner);
         } else if (psc.kind === "corporate-entity-beneficial-owner") {
           const beneficialOwnerOther = mapPscToBeneficialOwnerOther(psc);
+          beneficialOwnerOther.principal_address = mapPrivateAddress(boPrivateData, beneficialOwnerOther.ch_reference as string);
```

Comment:
Although it makes sense to include logic for the principal address of both OLE + Gov BOs, should we take them out for now so that they can be completed in the other tickets defined